src/options_app.c

```diff
@@ -0,0 +1,1479 @@
+        .priority = 0,
+        .areaOver = GX_BG_AREAOVER_XLU,
+        .mosaic = FALSE,
+    },
+};
+
+static const UnkStruct_0200D2B4 ov54_021E6EAC[9] = {
+    {
+        .resourceSet = 0,
+        .x = 112,
+        .y = 24,
+        .z = 0,
+        .animSeqNo = 0,
+        .rotation = 1,
+        .unk_10 = 0,
+        .whichScreen = NNS_G2D_VRAM_TYPE_2DMAIN,
+        .unk_18 = 0,
+        .unk_1C = 0,
+        .unk_20 = 0,
+        .unk_24 = 0,
+    },
+    {
+        .resourceSet = 0,
+        .x = 112,
+        .y = 48,
+        .z = 0,
+        .animSeqNo = 0,
+        .rotation = 1,
+        .unk_10 = 0,
+        .whichScreen = NNS_G2D_VRAM_TYPE_2DMAIN,
+        .unk_18 = 0,
+        .unk_1C = 0,
+        .unk_20 = 0,
+        .unk_24 = 0,
+    },
+    {
+        .resourceSet = 0,
+        .x = 112,
+        .y = 72,
+        .z = 0,
+        .animSeqNo = 1,
+        .rotation = 1,
+        .unk_10 = 0,
+        .whichScreen = NNS_G2D_VRAM_TYPE_2DMAIN,
+        .unk_18 = 0,
+        .unk_1C = 0,
+        .unk_20 = 0,
+        .unk_24 = 0,
+    },
+    {
+        .resourceSet = 0,
+        .x = 112,
+        .y = 96,
+        .z = 0,
+        .animSeqNo = 1,
+        .rotation = 1,
+        .unk_10 = 0,
+        .whichScreen = NNS_G2D_VRAM_TYPE_2DMAIN,
+        .unk_18 = 0,
+        .unk_1C = 0,
+        .unk_20 = 0,
+        .unk_24 = 0,
+    },
+    {
+        .resourceSet = 0,
+        .x = 112,
+        .y = 120,
+        .z = 0,
+        .animSeqNo = 1,
+        .rotation = 1,
+        .unk_10 = 0,
+        .whichScreen = NNS_G2D_VRAM_TYPE_2DMAIN,
+        .unk_18 = 0,
+        .unk_1C = 0,
+        .unk_20 = 0,
+        .unk_24 = 0,
+    },
+    {
+        .resourceSet = 1,
+        .x = 115,
+        .y = 144,
+        .z = 0,
+        .animSeqNo = 0,
+        .rotation = 1,
+        .unk_10 = 0,
+        .whichScreen = NNS_G2D_VRAM_TYPE_2DMAIN,
+        .unk_18 = 0,
+        .unk_1C = 0,
+        .unk_20 = 0,
+        .unk_24 = 0,
+    },
+    {
+        .resourceSet = 2,
+        .x = 213,
+        .y = 144,
+        .z = 0,
+        .animSeqNo = 0,
+        .rotation = 1,
+        .unk_10 = 0,
+        .whichScreen = NNS_G2D_VRAM_TYPE_2DMAIN,
+        .unk_18 = 0,
+        .unk_1C = 0,
+        .unk_20 = 0,
+        .unk_24 = 0,
+    },
+    {
+        .resourceSet = 3,
+        .x = 188,
+        .y = 170,
+        .z = 0,
+        .animSeqNo = 0,
+        .rotation = 1,
+        .unk_10 = 1,
+        .whichScreen = NNS_G2D_VRAM_TYPE_2DMAIN,
+        .unk_18 = 0,
+        .unk_1C = 0,
+        .unk_20 = 0,
+        .unk_24 = 0,
+    },
+    {
+        .resourceSet = 3,
+        .x = 116,
+        .y = 170,
+        .z = 0,
+        .animSeqNo = 0,
+        .rotation = 1,
+        .unk_10 = 1,
+        .whichScreen = NNS_G2D_VRAM_TYPE_2DMAIN,
+        .unk_18 = 0,
+        .unk_1C = 0,
+        .unk_20 = 0,
+        .unk_24 = 0,
+    },
+};
+
+#define wOptionsTitle          &data->windows[0]
```

Comment:
actually kinda unsure about the whole defines here....

Reply:
it kinda feels like rather than being an array of windows it'd benefit from being a struct of windows with defined names

Reply:
We have to use an array of windows here, since they're freed in a for-loop (see `OptionsApp_FreeWindows`)

Reply:
(ugly suggestion)
use a union?

Reply:
that works